### PR TITLE
New quickstart

### PR DESCRIPTION
**Pull request by strickvl**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by fa9r — COMMENTED** [review]
- love the showcase of new pipeline / steps features

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:636]: Minor nitpick but this looks a bit weird, maybe could be more readable if split into two lines

```python
versions = Client().active_stack.model_registry.list_model_versions()
most_recent_model_version_number = int(versions[0])
```

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:664]: Same here, maybe more intuitive to pass the params directly in the pipeline definition instead of using the `with_options` syntax

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:746]: Same as for the src/ definition, this `after` call is redundant since it's already required by the pipeline DAG
```suggestion
```

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:832]: Cool 👍

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:934]: Maybe the main CTA should now be to sign up to the cloud instead? What's the status of sandbox actually, isn't this deprecated in favor of the cloud now?

**Review by fa9r — APPROVED** [review]
Review part 2: Generally the new quickstart looks great, I think the use case is much more interesting than before, in particular the model selection step makes much more sense than the old "evaluator" step. Cloudflare also seems much easier to use than ngrok and I like your showcase of the new pipeline / steps features (running step separately, ...).

The only major request I have is that you test this thoroughly in Colab (once the pyyaml issue is resolved) and make sure everything (including dashboard/...) works end-to-end as expected with cloudfare etc. Other than that I left only minor suggestions, mostly on how to make the step/pipeline definitions a bit easier to read/understand.

Will already pre-approve this with the assumption that you have tested it in Colab before clicking the big green merge button :)

File: examples/quickstart/README.md
Comment (strickvl) [reply examples/quickstart/README.md:30]: Not really sure what the instruction should be changed to. We haven't removed
the examples CLI command yet, nor have we removed / updated the examples
directory yet etc. I think the default would be to leave this as it is until
those other two pieces change. There will in any case be an effort to remove
those comments from READMEs etc later, no? WDYT?

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (strickvl) [reply examples/quickstart/notebooks/quickstart.ipynb:46]: Yes it works on colab without those. At least with the current ZenML version...

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (strickvl) [reply examples/quickstart/notebooks/quickstart.ipynb]: Good catch. I moved it down where it's actually commenting on one of those `after` calls.

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (strickvl) [reply examples/quickstart/notebooks/quickstart.ipynb:933]: Unfortunately the metadata is a required input, even if it's empty. I'll make a ticket, but @safoinme confirmed that we can make it optional.

Comment (strickvl) [reply examples/quickstart/notebooks/quickstart.ipynb:933]: https://zenml.atlassian.net/browse/FRW-2163

File: examples/quickstart/notebooks/quickstart.ipynb
Comment (strickvl) [reply examples/quickstart/notebooks/quickstart.ipynb]: Written before the cloud was a thing :) I've updated now so that everything points to the cloud.

File: examples/quickstart/README.md
Comment (fa9r) [reply examples/quickstart/README.md:29]: I don't think there will be an such an effort since we're removing all other examples anyways, so only quickstart needs to be adjusted actually. I would just say "clone ZenML and go into examples/quickstart" instead of the `zenml example ...` instructions.

**Pull request merged**
